hack/periodic/content-update.sh

```diff
@@ -16,7 +16,7 @@ git reset HEAD^
 curl -u openshift-azure-robot:$GITHUB_TOKEN -H "Content-Type:application/json" -d @- -so /dev/null https://api.github.com/repos/openshift/openshift-azure/pulls <<'EOF'
 {
     "title": "Content update",
-    "body": "```release-notes\nNONE\n```",
+    "body": "```release-note\r\nNONE\r\n```",
```

Comment:
did you mean to take the "s" off the release-notes?

Reply:
should be 
```suggestion
    "body": "```release-notes\r\nNONE\r\n```",
```